AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -588,6 +588,7 @@ sravnibank.com.ua##a[href="https://sravnibank.com.ua/kredity-nalichnymi/"]
 wotexpress.info##.twitch_prime_june
 artstation.com#%#//scriptlet('remove-class', 'is-promobar-active', 'body')
 cashtocode.com##.promo-area
+mobiflip.de#?#ul.infinite-content > div > li:has(> div > a[href^="https://www.mobiflip.de/go"])
```

Comment:
Please move rule up (near https://github.com/AdguardTeam/AdguardFilters/issues/59909)  and include link to issue.

Reply:
No need to add issue link in this file, if the rule looks obvious